README.md

```diff
@@ -14,16 +14,44 @@ The protocol is described in detail in the [Morpho Blue Whitepaper](./morpho-blu
 ## Getting Started
 
 > [!IMPORTANT]
-> It is advised to use a dedicated address whose only purpose is to deploy all contracts associated with Blue on each EVM-compatible chain, so that addresses are common across chains.
+> It is advised to use a dedicated address whose only **lifetime** purpose is to deploy all contracts associated with Blue on each EVM-compatible chain, so that addresses are common across chains.
```

Comment:
which address are you referring to? The deployed one right? Not the deployer

Reply:
```suggestion
> It is advised to use a dedicated address whose only **lifetime** purpose is to deploy all contracts associated with Blue on each EVM-compatible chain, so that deployed contract addresses are common across chains.
```

Reply:
this is valid only for non create 2 deployment. Not sure it's necessary to say that then

Reply:
It is for oracles and IRM

Reply:
Or we deploy everything using CREATE2

Reply:
for the IRM it's ok, for oracles we don't care i think

Reply:
done in #17